CHANGELOG.md

```diff
@@ -2,6 +2,19 @@
 
 ## [next]
 
+### Added
+- Added environment variables: `LAN_ONLY` and `MAX_SNAPSHOTS`.
```

Comment:
Please use present tense here and remove the colon, like "Add environment variables `LAN_ONLY` and `MAX_SNAPSHOTS`.".